packages/ses/test/test-repair-intrinsics.js

```diff
@@ -28,6 +37,14 @@ test('permitted prototypes - on', t => {
   hardenIntrinsics();
   console.timeEnd('Benchmark hardenIntrinsics()');
 
+  // eslint-disable-next-line no-eval
+  t.truthy(globalThis.eval && Object.isFrozen(globalThis.eval));
+  t.truthy(globalThis.Function && Object.isFrozen(globalThis.Function));
+  t.truthy(globalThis.Compartment && Object.isFrozen(globalThis.Compartment));
+  for (const prop of getOwnPropertyNames(initialGlobalPropertyNames)) {
+    globalThis[prop] && t.truthy(Object.isFrozen(globalThis[prop]));
+  }
```

Comment:
The iterand is the name of the property, whereas prop could just as well imply the corresponding property descriptor or property value. I moved the outer condition so that the assertion count is consistent regardless of whether `&&` short-circuits evaluation.

```suggestion
  for (const name of getOwnPropertyNames(initialGlobalPropertyNames)) {
    t.ok(globalThis[name] && Object.isFrozen(globalThis[name]));
  }
```

Reply:
fixed 4bef5ee26